### PR TITLE
Edmundson Ch. 7

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: journal/modernized/08-chapter-7.adoc [hunk 1]
```diff
@@ -224,10 +224,10 @@ And thus the warning before given them was fulfilled.
 
 In the year 1609, George Fox came to Ireland,
 and several Friends in the ministry with him.
-Me settled men`'s and women`'s meetings among Friends throughout the nation,
+He settled men`'s and women`'s meetings among Friends throughout the nation,
 that faithful men and women should take care in the government of church affairs,
 among our own Society, which meetings are of great service.
-I was much cased by them, as I told George Fox at that time,
+I was much eased by it, as I told George Fox at that time,
```
Comment (maijapapaya) [inline journal/modernized/08-chapter-7.adoc:230]: So when I went to look up "cased" in the PDF, I noticed "them " is actually supposed to be "it". Just wanted you to know why there was such a big change. Also, I accidentally tapped some keys where I could use emojis but i have no idea how I did it! Can you tell me how?? :)

Comment (jaredh159) [reply journal/modernized/08-chapter-7.adoc:230]: Got it on the change, makes a lot more sense than "cased by them". Good catch. 👍

About emojis, you basically type a _colon_ and then it pops up an auto-complete dropdown, and you can type to get access to all of the emojis, and use a combination of your typing and arrow keys to pick one, and then press “Tab” to select (I put a screenshot thing in the asciidoc channel as well)

File: journal/modernized/08-chapter-7.adoc [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
-Of the government, and Chancellor Boyle`'s clemency towards Friends,
-upon William Edmundson`'s petitioning them against G. Clapham`'s cruelty,
-then priest of Mount-melick.--Of his prophetic warning to the inhabitants of Londonderry;
-and conference afterwards with the bishop and mayor of that place;
-and the settling of meetings for Discipline in Ireland, by George Fox, in 1669.
+[.chapter-synopsis]
+* The Government, and Chancellor Boyle`'s Clemency Towards Friends Upon William Edmundson`'s Petitioning Them Against G. Clapham`'s Cruelty, Then Priest of Mount-melick
+* His Prophetic Warning To the Inhabitants of Londonderry; and Conference Afterwards With the Bishop and Mayor of That Place
+* The Settling of Meetings For Discipline In Ireland, By George Fox, In 1669
+* //I'm wondering if I should shorten these up at all? Not totally comfortable doing that yet.
```
Comment (jaredh159) [inline journal/modernized/08-chapter-7.adoc]: This is tough, agreed. If it makes you feel any better, I've never seen another book that needed this much massaging of the chapter-synopsis, so this is not very typical. Here's what I would propose (just a slight alteration from what you've got here):

```ASCIIDOC
[.chapter-synopsis]
* The government and Chancellor Boyle`'s clemency towards Friends, upon William Edmundson`'s petitioning them against G. Clapham`'s cruelty, then priest of Mount-melick
* Prophetic warning to the inhabitants of Londonderry
* Conference afterwards with the bishop and mayor of that place
* Settling of meetings for Discipline in Ireland, by George Fox, in 1669
```

You can copy/paste that block and use it for both editions.

Note that I didn't capitalize them -- I mentioned this in our slack DM yesterday but it was in the middle of a bunch of other stuff, so not sure if you saw it -- you don't need to keep capitalizing these, I realized I could do the capitalization programmatically as part of the computer's process of turning the asciidoc into pdfs and ebooks. So, there's no need to change back the old ones, but you can skip the step of capitalizing from here on out.

File: journal/original/08-chapter-7.adoc [hunk 3]
```diff
@@ -1,10 +1,9 @@
...
 
-Of the government, and Chancellor Boyle`'s clemency towards Friends,
-upon William Edmundson`'s petitioning them against G. Clapham`'s cruelty,
-then priest of Mount-melick.--Of his prophetic warning to the inhabitants of Londonderry;
-and conference afterwards with the bishop and mayor of that place;
-and the settling of meetings for Discipline in Ireland, by George Fox, in 1669.
+[.chapter-synopsis]
+* The Government, and Chancellor Boyle`'s Clemency Towards Friends Upon William Edmundson`'s Petitioning Them Against G. Clapham`'s Cruelty, Then Priest of Mount-melick
+* His Prophetic Warning To the Inhabitants of Londonderry; and Conference Afterwards With the Bishop and Mayor of That Place
+* The Settling of Meetings For Discipline In Ireland, By George Fox, In 1669
```
Comment (jaredh159) [inline journal/original/08-chapter-7.adoc]: Let's use the modification I made in the other comment here as well.

**Review by jaredh159 — CHANGES_REQUESTED** [review]
Nice work, left a comment on an alteration of the chapter synopsis, but everything else looks great!

File: journal/modernized/08-chapter-7.adoc (hunk 1 above)
Comment (maijapapaya) [reply journal/modernized/08-chapter-7.adoc:230]: 👍 Thanks!!

File: journal/modernized/08-chapter-7.adoc (hunk 2 above)
Comment (maijapapaya) [reply journal/modernized/08-chapter-7.adoc]: Yes, i think I had done that before you wrote the message. I had a couple of things to finish up on that by looking at the PDF, so it wasn't sent right away.

**Review by jaredh159 — APPROVED** [review]

**Comment by jaredh159** [comment]
great job, thanks! merging!

**Pull request merged**